transports/wasm-ext/src/lib.rs

```diff
@@ -40,11 +40,17 @@ use libp2p_core::{
     transport::{ListenerId, TransportError, TransportEvent},
     Multiaddr, Transport,
 };
-use parity_send_wrapper::SendWrapper;
+use send_wrapper::SendWrapper;
```

Comment:
👍

Reply:
When I split my changes to a different package, do you want this change in `libp2p-wasm-ext`?